example/src/Http.tsx

```diff
@@ -2,14 +2,15 @@ import * as React from 'react';
 import { SafeAreaView, StyleSheet, ScrollView } from 'react-native';
 import Rive from 'rive-react-native';
 
+const url = 'https://cdn.rive.app/animations/juice_v7.riv';
```

Comment:
I see how these lines that are commented out might be very helpful for testing our solutions so I guess you want to leave them for now?